src/app/visual/autonumeric/autonumeric-visual.component.html

```diff
@@ -102,11 +139,21 @@ <h2>
     Set options by preset
   </button>
 
+  <button
+    class="sky-btn"
+    type="button"
+    (click)="setOptionsToCustom()"
+  >
+    Set options to custom
+  </button>
+
   <button
     class="sky-btn"
     type="button"
     (click)="onDisableClick()"
   >
     Toggle disabled
   </button>
+
+  <pre>{{ autonumericOptions | json }}</pre>
 </div>
```

Comment:
When I click the 'Set Value' button in the visual demo, a console error gets thrown: 
```
ERROR Error: Must supply a value for form control with name: 'currency'.
```